package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -449,19 +452,39 @@ def write(self, selection, frame=None):
 class DumpReader(base.ReaderBase):
     """Reads the default `LAMMPS dump format`_
 
-    Expects trajectories produced by the default 'atom' style dump.
+    Supports coordinates in the LAMMPS "unscaled" (x,y,z), "scaled" (xs,ys,zs),
+    "unwrapped" (xu,yu,zu) and "scaled_unwrapped" (xsu,ysu,zsu) coordinate
+    conventions (see https://docs.lammps.org/dump.html for more details).
+    If no coordinate convention is provided, or if "auto" is selected,
+    one will be guessed. Auto proceeds in the order "unscaled",
+    "scaled", "unwrapped", "scaled_unwrapped" and whichever set of coordinates
+    is detected first will be used. If coordinates are given in the scaled
+    coordinate convention (xs,ys,zs) or scaled unwrapped coordinate convention
+    (xsu,ysu,zsu) they will automatically be converted from their
+    scaled/fractional representation to their real values.
 
-    Will automatically convert positions from their scaled/fractional
-    representation to their real values.
 
+    .. versionchanged:: 2.0.0
+       Now parses coordinates in multiple lammps conventions (x,xs,xu,xsu)
     .. versionadded:: 0.19.0
     """
     format = 'LAMMPSDUMP'
+    _conventions = ["auto", "unscaled", "scaled", "unwrapped",
+                    "scaled_unwrapped"]
 
-    def __init__(self, filename, **kwargs):
+    def __init__(self, filename, lammps_coordinate_convention="auto",
+                 **kwargs):
         super(DumpReader, self).__init__(filename, **kwargs)
 
         root, ext = os.path.splitext(self.filename)
+        if lammps_coordinate_convention in self._conventions:
+            self.lammps_coordinate_convention = lammps_coordinate_convention
+        else:
+            raise ValueError("coordinate convention"
+                             f"{lammps_coordinate_convention} incorrectly "
+                             "specified, please specify one of auto, unscaled"
+                             "scaled, unwrapped or scaled_unwrapped")
```

Comment:
I think this will result in "coordinate **conventionBADCONVENTION** incorrectly specified, please specify one of auto, **unscaledscaled**, unwrapped or scaled_unwrapped"

I bolded the missing spaces. Could you please also mention the `lammps_coordinate_convention` keyword in the error message and make it clear that the options are strings, like the below?

```python
			option_string = "'auto', '" + "', '".join(COORDTYPE_COLUMN_NAMES) + "'"
            raise ValueError("lammps_coordinate_convention="
							 f"'{lammps_coordinate_convention}' is not a valid option. "
							 f"Please choose one of {option_string}"
```

Reply:
okay :+1: